data/mods/Aftershock/mobs/aliens.json

```
@@ -151,5 +151,41 @@
     "anger_triggers": [ "STALK", "FRIEND_ATTACKED", "FRIEND_DIED", "PLAYER_WEAK", "PLAYER_CLOSE" ],
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "afs_titanis",
+    "name": { "str": "titanis" },
+    "species": "BIRD",
+    "default_faction": "bear",
+    "bodytype": "ostrich",
+    "symbol": "T",
+    "color": "blue_green",
+    "volume": "92500 ml",
+    "weight": "120000 g",
+    "material": "flesh",
+    "aggression": -10,
```

```suggestion
    "aggression": -10,
```
For some reason I thought that aggression lower than three means it will never become hostile?  I could be very wrong on that thought though.

---

It will still attack small animals and bees (HATE toward those factions). Regarding the character, from my reading of monster.cpp and other knowledge, assuming no character traits like Animal Empath or Animal Discord, unless both of the anger triggers are going off (to a total of 20 or more) it'll ignore them if at full HP, or flee if not. If both triggers do go off, it will attack.

---

Right, I literally copied bear aggression because that seems in gameplay to hit the right balance of menace but not constant aggression.